app/controllers/configurations_controller.rb

```diff
@@ -19,20 +19,20 @@ def index
   def create
     respond_to do |format|
       format.html { head :method_not_allowed }
-      
+
       if @config.save
         headers['Location'] = configuration_url(@config)
-        
-        format.xml { 
-          render :xml => @config.to_xml, :status => :created 
+
+        format.xml {
+          render :xml => @config.to_xml, :status => :created
         }
         format.js {
           render :json => @config.to_json, :status => :created
         }
       else
-        format.xml { 
-          render :xml => @config.errors.to_xml, 
-          :status => :unprocessable_entity 
+        format.xml {
+          render :xml => @config.errors.to_xml,
+          :status => :unprocessable_entity
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.